pkg/apis/v1/ec2nodeclass_status.go

```diff
@@ -64,11 +64,11 @@ type AMI struct {
 
 // EC2NodeClassStatus contains the resolved state of the EC2NodeClass
 type EC2NodeClassStatus struct {
-	// Subnets contains the current Subnet values that are available to the
+	// Subnets contains the current Subnets values that are available to the
```

Comment:
```suggestion
	// Subnets contains the current subnet values that are available to the
```

I think we should leave this as-is. The old grammar was correct